src/bin/nydusd/blob_cache.rs

```diff
@@ -179,6 +179,7 @@ impl BlobCacheMgr {
                 .insert(config.get_key().to_string(), config);
             // Try to add the referenced data blob object if it doesn't exist yet.
             for bi in rs.superblock.get_blob_infos() {
+                info!("Found blob {} on domain {}", &bi.blob_id(), domain_id);
```

Comment:
Should we remove this info? Or change it to debug?

Reply:
changing info debug makes sense to me, will do.

Reply:
done.